msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -1454,6 +1452,17 @@ When you use these install and uninstall events, there are some instances where
 
 It's important to know that new events can be added anytime in the future and your bot begins to receive them. So you must design for the possibility of receiving unexpected events. If you're using the Bot Framework SDK, your bot automatically responds with a 200 – OK to any events you don't choose to handle.
 
+## Handling errors in conversation events
+
+When the bot encounters an error while handling different events or activities, it shouldn't post random messages directly to the conversation, following is an example:
+
+:::image type="content" source="../../../assets/images/error-msg.PNG" alt-text="The screenshot shows an error message to the conversation with no meaningful context.":::
+
+The bot must send meaningful messages in conversations with specific error message for better user experience. For more information, see [handle unexpected errors](/azure/bot-service/bot-builder-howto-handle-user-interrupt).
```

Comment:
Frankly the problem is not necessarily lack of context, but the message itself. I'd keep this simple and discourage responding to conversationUpdate messages _at all_. Welcome messages should be handled via installationUpdate.

Reply:
There's a similar problem for invoke activities.

Reply:
@v-sdhakshina - Update it like below:
The bot must send meaningful messages in conversations with specific events like:
Welcome messages should be handled via [installationUpdate](https://docs.microsoft.com/en-us/microsoftteams/platform/bots/how-to/conversations/subscribe-to-conversation-events?tabs=dotnet#installation-update-event).

Reply:
updated the document as suggested.